Add Zivid theme

knatten opened the pull request:
Based on the "white" theme, but:
- Use Montserrat font for slides
- Use Atom One Light color scheme for code
- Don't use upper case headers
- Zivid logo in the background
- Set Zivid colours for links, bullet points, sub headers
- Style table with alternating row colours

Had to enable customisation of a few more things in reveal.js for this
to work

knatten committed:
Add Zivid theme

Based on the "white" theme, but:
- Use Montserrat font for slides
- Use Atom One Light color scheme for code
- Don't use upper case headers
- Zivid logo in the background
- Set Zivid colours for links, bullet points, sub headers
- Style table with alternating row colours

Had to enable customisation of a few more things in reveal.js for this
to work

knzivid commented:
- What are the changes to `dist/theme/beige.css`, `dist/theme/blood.css` and so on for? Can we remove those themes from this repo? The table font color and background are both white with the blood theme.
- Can you minify the png? 512Kb is a lot :D

knatten commented:
> * What are the changes to `dist/theme/beige.css`, `dist/theme/blood.css` and so on for? Can we remove those themes from this repo? The table font color and background are both white with the blood theme.

The changes are  because I extended the customisation points for reveal.js in `settings.scss`, then it adds those to the generated css files for each theme. The problem with the blood theme was due to a bug in `settings.scss`, which I fixed in 471916a7ef56 now.

I think it will be easier to pull in changes from upstream if we don't delete the other themes. 

> * Can you minify the png? 512Kb is a lot :D

Good point. Switched to svg in 542cda61e6, 1391 bytes is better!

knatten commented:
RFNR
